common/COMMON_moveReportToRSlims.py

```diff
@@ -34,29 +35,33 @@ def main():
     # For each output artifact, download the attached PDF and move to the RS LIMS file surface.
     for action in inart_actions:
         inart = action["artifact"]
+        # Action gives a string with the choosen next step for the artifact. This is the 
+        # last step in the protocol so it is complete, repeat or so on.
         # Only move report if sample is marked as complete
         if not "complete" in action["action"]:
             logging.info(f"Sample not complete, report not moved, {inart.name}")
-        else:
-            outart = get_outart(outarts, inart)
-            if not outart.files:
-                logging.info(f"No report for {outart.name}")
-            else:
-                # Get the name of the file in Clarity LIMS
-                filename = outart.files[0].original_location.split("/")[-1]
-                # Create the path to the RS LIMS folder
-                path = RS_LIMS_PAHT + filename
-                # Get the file as a readcontent object
-                read_content = lims.get_file_contents(uri=outart.files[0].uri)
+            continue
+
+        outart = get_outart(outarts, inart)
+        if not outart.files:
+            logging.info(f"No report for {outart.name}")
+            continue
 
-                project = outart.samples[0].project
-                if project.udf.get("Project Classification", "") == "Laborativ - Klinisk":
-                    # Write binary report
-                    with open(path, "wb") as f:
-                        f.write(read_content.read())
-                    logging.info(f"Report moved to RS Lims, {filename}")
-                else:
-                    logging.info(f"Test sample, report not moved, {outart.name}")
+        # Get the name of the file in Clarity LIMS
+        filename = outart.files[0].original_location.split("/")[-1]
```

Comment:
```suggestion
        filename = os.path.basename(outart.files[0].original_location)
```

since`os` is already imported in this module